### PR TITLE
Added badges + refresh table of content (automatically)

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Added badges + refresh table of content (automatically)

## Type of change

- Documentation update

## Testing steps

N/A

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,67 +36,67 @@ Utilities for Insights Results Aggregator
...
         * [Description](#description-13)
-        * [Generated documentatin](#generated-documentatin-1)
+        * [Generated documentatin](#generated-documentatin)
         * [Usage](#usage-9)
-* [Utilities for working with objects stored in AWS S3 bucket](#utilities-for-working-with-objects-stored-in-aws-s3-bucket)
-    * [`upload_timestamps.py`](#upload_timestampspy)
+    * [`cluster_results_age.py`](#cluster_results_agepy)
         * [Description](#description-14)
-        * [Generated documentation](#generated-documentation-8)
+        * [Generated documentatin](#generated-documentatin-1)
```
Comment (epapbak) [inline README.md:70]: where's the typo coming from?

Comment (tisnik) [reply README.md]: from text which is source of truth. fixed there, good catch

File: README.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # insights-results-aggregator-utils
 Utilities for Insights Results Aggregator
 
+[![made-with-python](https://img.shields.io/badge/Made%20with-Python-1f425f.svg)](https://www.python.org/)
+[![forthebadge made-with-go](http://ForTheBadge.com/images/badges/made-with-go.svg)](https://go.dev/)
```
Comment (juandspy) [inline README.md:5]: I'd use [forthebadge made-with-go](https://go.dev/) as the other badge is quite different and losses the aesthetic of the repo :smile:

**Review by juandspy — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (tisnik) [reply README.md:5]: because Go is better :d

Comment (tisnik) [reply README.md]: updated

**Pull request merged**